src/components/FileList/__tests__/FileElement.test.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import {MemoryRouter} from 'react-router-dom';
+import {render} from 'react-testing-library';
+import FileElement from '../FileElement';
+import studyByKfId from './studyByKfId';
```

Comment:
maybe we can move mock data somewhere globally accessible in the future

Reply:
planning to figure this out during the functional testing.
Looking at using a mocked apollo provider with a common set of static queries loaded from some directory: https://www.apollographql.com/docs/apollo-server/features/mocking